ecllibrary/teststd/Date/TestDate.ecl

```diff
@@ -435,8 +435,7 @@ EXPORT TestDate := MODULE
     END;
 
     x := LOOP(firstDate, 1000000, processNextDate(ROWS(LEFT)));
-
-    EXPORT Test01 := OUTPUT(x);
+    EXPORT Test01 := ASSERT( x[1].day = 1000001 AND x[1].gregorian.year = 2738 AND x[1].gregorian.month = 11 AND x[1].gregorian.day = 29 AND x[1].julian.year = 2738 AND x[1].julian.month = 11 AND x[1].julian.day = 8, 'Failed') ;
```

Comment:
minor: Would benefit of being split over multiple lines.

Reply:
Something like this?